insights/parsers/dnf_module.py

```diff
@@ -93,8 +154,24 @@ def parse_content(self, content):
                         trailing_ignore=['Hint:', 'Error:'])
         if not data:
             raise SkipComponent('Nothing need to parse.')
-        for m in data:
-            self.update({m['Name']: DnfModuleBrief(m)})
+
+        modules_streams = {}
+        for ms in data:
+            if ms['Name'] in modules_streams:
+                modules_streams[ms['Name']].add_stream(ms)
```

Comment:
Is `Name` key guaranteed to be present?

Reply:
yes, it is always printed by `dnf module list`